lib/server.js

```diff
@@ -19,30 +19,50 @@ util.buildIndex(isDist); // throw compilation errors right away
 let app = express();
 let env = util.readEnv(isDist);
 let port = parseInt(process.env.PORT || 4300);
-app.use(morgan('combined', { skip: req => !util.isIndex(req.path) }));
+app.use(morgan('combined', {
+  skip: req => req.path === '/' || !util.isIndex(req.path)
```

Comment:
Stopped logging root-path requests, which should get all the ALB health checks out of the logs.